tuf/tuf.go

```diff
@@ -771,3 +771,19 @@ func (tr Repo) sign(signedData *data.Signed, role data.Role) (*data.Signed, erro
 	}
 	return signedData, nil
 }
+
+// GetRole returns a RoleWithKeys object, given a role name.
+func (tr Repo) GetRole(role string) (*data.RoleWithKeys, error) {
```

Comment:
nit (especially since this may change), but it might be nice to name this `GetRoleWithKeys`